hashing/hasher.go

```diff
@@ -1,5 +1,8 @@
 package hashing
 
+// BlsHashSize specifies the hash size for using bls scheme
+const BlsHashSize = 16
```

Comment:
This is related with the multisig aggregation step. So it should be placed somewhere in the crypto/signing/multisig package

Reply:
ok